src/EventSubscriber/GatedContentSubscriber.php

```diff
@@ -64,7 +64,12 @@ public function accessCheck(GetResponseEvent $event) {
         break;
     }
 
-    $entity_types = ['gc_video', 'vy_blog_post', 'live_stream'];
+    $entity_types = [
+      'gc_video',
```

Comment:
As far as I understand these are entity bundles. Shouldn't it also contain entity types?

Reply:
Yes, correct, this is bundles.
Entity types selected here:
```php
    $route_object = NULL;
    switch ($route_name) {
      case 'entity.node.canonical':
        $route_object = $request->get('node');
        break;

      case 'entity.eventseries.canonical':
        $route_object = $request->get('eventseries');
        break;

      case 'entity.eventinstance.canonical':
        $route_object = $request->get('eventinstance');
        break;
    }
```